ipld/retriever_test.go

```diff
@@ -94,3 +94,33 @@ func TestRetriever_ByzantineError(t *testing.T) {
 	var errByz *ErrByzantine
 	require.ErrorAs(t, err, &errByz)
 }
+
+// TestRetriever_MultipleRandQuadrants asserts that reconstruction succeeds
+// when any three random quadrants requested.
+func TestRetriever_MultipleRandQuadrants(t *testing.T) {
+	RetrieveQuadrantTimeout = time.Millisecond * 500
```

Comment:
If this global var is used in multiple tests, this assignment here will have side-effects.

Reply:
Yeah, but I am controlling that. We have https://github.com/celestiaorg/celestia-node/issues/709 to fix this.